Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -48,6 +48,9 @@
   --chromium-border-radius: 1px;
   --mozilla-box-shadow: 0 0 0 2px royalblue;
   --mozilla-border-radius: 1px;
+  --remotion-themed-font-heading: normal 700 14px Arial, 'Helvetica', sans-serif;
+  --remotion-themed-font-bold: normal 500 14px Arial, 'Helvetica', sans-serif;
+  --remotion-themed-font-standard: normal 400 14px Arial, 'Helvetica', sans-serif;
```

Comment:
If I understood mike correctly, this should only apply the font weight and inherit the rest. `style.css` should override those with the fonts that we want.

Reply:
correct, if we can do this that way. if we need to name the font, maybe we can substitute `inherit` or `auto`, otherwise, go with just `sans-serif` for the font type. and the size is set to `100%` or `inherit`